src/Tools/ResponseResolver.php

```diff
@@ -22,23 +26,49 @@ class ResponseResolver
      */
     private $route;
 
+    /**
+     * @param Route $route
+     */
     public function __construct(Route $route)
     {
         $this->route = $route;
     }
 
+    /**
+     * @param array $tags
+     * @param array $routeProps
+     *
+     * @return array
+     */
     private function resolve(array $tags, array $routeProps)
     {
         $response = null;
+
         foreach (static::$strategies as $strategy) {
             $strategy = new $strategy();
+
+            /** @var JsonResponse|array|null $response */
             $response = $strategy($this->route, $tags, $routeProps);
+
             if (! is_null($response)) {
-                return $this->getResponseContent($response);
+                if (is_array($response)) {
+                    return array_map(function (JsonResponse $response) {
+                        return ['status' => $response->getStatusCode(), 'content' => $this->getResponseContent($response)];
+                    }, $response);
+                }
+
+                return [['status' => $response->getStatusCode(), 'content' => $this->getResponseContent($response)]];
```

Comment:
In line with the above changes, this would become:

```
                return [['status' => $responses->getStatusCode(), 'content' => $this->getResponseContent($responses)]];
```

But I would prefer that the strategies **always** returned an array of responses. Much better for consistency.

So let's have the TransformerTagsStrategy and ResponseCallStrategy wrap their response in an array. Then we can get rid of this line, and the `if (is_array...`

Reply:
Yep, that's a good point. Done.